driver-core/src/main/java/com/datastax/driver/core/SchemaParser.java

```diff
@@ -491,9 +504,13 @@ private static ResultSet get(ResultSetFuture future) throws InterruptedException
         final Map<String, List<Row>> aggregates;
         final Map<String, List<Row>> views;
         final Map<String, Map<String, List<Row>>> indexes;
+        final ResultSet virtualKeyspaces;
+        final Map<String, List<Row>> virtualTables;
+        final Map<String, Map<String, Map<String, ColumnMetadata.Raw>>> virtualColumns;
 
         public SystemRows(ResultSet keyspaces, Map<String, List<Row>> tables, Map<String, Map<String, Map<String, ColumnMetadata.Raw>>> columns, Map<String, List<Row>> udts, Map<String, List<Row>> functions,
-                          Map<String, List<Row>> aggregates, Map<String, List<Row>> views, Map<String, Map<String, List<Row>>> indexes) {
+                          Map<String, List<Row>> aggregates, Map<String, List<Row>> views, Map<String, Map<String, List<Row>>> indexes, ResultSet virtualKeyspaces, Map<String, List<Row>> virtualTables,
+                          Map<String, Map<String, Map<String, ColumnMetadata.Raw>>> virtualcolumns) {
```

Comment:
nit: I see `virtualcolumns` was renamed `virtualColumns` for the class member, but the constructor parameter is still `virtualcolumns`.